transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -218,6 +220,12 @@ public B validate() {
         return (B) this;
     }
 
+    private void ensureChannelFactory() {
+        if (channelFactory == null) {
+            channelFactory = ChannelFactoriesRegistry.getFactoryForEventLoopGroup(group.getClass(), isServerSide());
```

Comment:
yep. I updated comment above. 
If `getDefaultChannelFactory()` returns `null`, this will lead to an runtime NPE in the users subclusses.